charts/rstudio-workbench/test-values.yaml

```diff
@@ -0,0 +1,17 @@
+# Configures user home directory shared storage
```

Comment:
@SamEdwardes, it looks like you committed your test file.

Reply:
Ooos sorry about that! I will fix.